src/modules/commander/Commander.cpp

```diff
@@ -1787,6 +1791,34 @@ Commander::run()
 			if (!_arm_state_machine.isArmed()) {
 				updateParameters();
 
+#if PX4_EXPORT_RESTRICTION_BUILD
```

Comment:
It should be #ifdef in case there's no explicit boolean condition afterwards. no?
```suggestion
#ifdef PX4_EXPORT_RESTRICTION_BUILD
```